### PR TITLE
VideoCapture add open/read timeout params to FFMPEG backend

### PR DESCRIPTION
**Pull request by iyadahmed**
A little feature that was merged recently to 3.4

port of #20541

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [x] The PR is proposed to proper branch

### CODE REVIEW

File: modules/videoio/src/cap_ffmpeg_impl.hpp [hunk 1]
```diff
@@ -928,6 +935,16 @@ bool CvCapture_FFMPEG::open(const char* _filename, const VideoCaptureParameters&
         if (params.has(CAP_PROP_HW_ACCELERATION_USE_OPENCL)) {
             use_opencl = params.get<int>(CAP_PROP_HW_ACCELERATION_USE_OPENCL);
         }
+#if USE_AV_INTERRUPT_CALLBACK
+        if (params.has(CAP_PROP_OPEN_TIMEOUT))
```
Comment (alalek) [inline modules/videoio/src/cap_ffmpeg_impl.hpp:939]: `_MSEC` is missing here

Comment (iyadahmed) [reply modules/videoio/src/cap_ffmpeg_impl.hpp:939]: done

**Comment by iyadahmed** [comment]
should be fixed now

**Comment by iyadahmed** [comment]
Checks passed

**Review by alalek — APPROVED** [review]
Thank you :+1:

**Pull request merged**